package.json

```diff
@@ -101,7 +101,6 @@
     "babel-polyfill": "6.20.0",
     "babel-register": "6.18.0",
     "bluebird": "2.9.34",
-    "body-parser": "1.17.2",
```

Comment:
__note:__ removing all ExpressJS specific packages, we can easily return them back when needed.

Reply:
Can you explain this more? If we are still using ExpressJS, why would we remove these?

Reply:
Once this PR lands we'll be using only Hapi in new platform, hence we don't need any ExpressJS packages.